dev/df_new_loader.py

```diff
@@ -2,12 +2,14 @@
 ##############################################################
 #
 # This script is used to test the dataframe of the new loader
```

Comment:
We may want to move this script to a proper unit_test in the future :-)

Reply:
This script will be obsolete soon when the new features are integrated in the pipeline following #596.